site/content/docs/01-component-format.md

```diff
@@ -42,7 +42,7 @@ Svelte uses the `export` keyword to mark a variable declaration as a *property*
 
 ---
 
-You can specify a default value, which will be used if the component's consumer doesn't specify a prop.
+You can specify a default value for a prop. It will be used if the component's consumer doesn't specify the prop on the component. Note that whenever a specified prop is removed by the consumer, the value is set to `undefined` rather than the default value.
```

Comment:
Given the current behaviour I'd prefer we don't use the term `default` anymore as it can be misleading.